src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ContributorInviteRequestViewModel.cs

```diff
@@ -0,0 +1,14 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
+{
+    public class ContributorInviteRequestViewModel
```

Comment:
Could simplify this to either `ContributorInviteRequest` or `ContributorInviteViewModel`. Wouldn't combine them both.

Can appreciate that our naming conventions around this are basically a mess, but this isn't really a view model because it's not going to be featured in responses. On the otherhand, it could be argued that we might consider everything at the HTTP boundaries (incoming or outgoing) to be a 'view model'. Either way, wouldn't make the situation even worse by combining the two conventions.

Not sure about how others feel about this @benoutram @duncan-at-hiveit, but feels like we need to get a handle on these naming conventions! Personally, I'd probably lean towards splitting out a `Requests` namespace that specifically contains requests.

Reply:
I changed `ContributorInviteRequest` to `ContributorInviteRequestViewModel` in response to another PR comment. I've left a comment about this in Slack and will change it when a consensus is reached.

Reply:
Consensus was to use `BlahViewModel` moving forward. I'll make this change to all other instances.